frontend/app/stores/toolStore.ts

```diff
@@ -93,7 +94,8 @@ export const toolState = proxy({
   isWalletConnected: false,
   hasRemoteConfigs: false,
   walletConnectStep: 'unfilled' as StepStatus,
-  buildStep: 'unfilled' as StepStatus
+  buildStep: 'unfilled' as StepStatus,
+  banner: createBannerStore() as BannerStore
```

Comment:
I'd suggest keeping bannerStore and toolStore separate even at this stage. It'll help us keep all stores simplified and disconnected.